docs/source/notebooks/iv_pymc.ipynb

```diff
@@ -0,0 +1,4154 @@
+{
```

Comment:
<p>In the final example, let's check that printing out the idata renders ok. If not, skip this or just selectively look if there's something specific</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Had to shorten some of the idata summary tables as they were projecting out too far beyond the border. Removed idata object in general: 
<img width="730" alt="image" src="https://github.com/pymc-labs/CausalPy/assets/10588167/926ab380-3c0d-47f1-886c-97a2ebd6b47c">